Explorer/Assets/DCL/CharacterPreview/ICharacterPreviewFactory.cs

```diff
@@ -0,0 +1,23 @@
+using Arch.Core;
+using DCL.Optimization.Pools;
+using UnityEngine;
+
+namespace DCL.CharacterPreview
+{    /// <summary>--
+    ///     Check ICharacterPreviewFactory in the old renderer
+    /// </summary>
+    public interface ICharacterPreviewFactory
+    {
+        CharacterPreviewController Create(World world, IComponentPoolsRegistry poolsRegistry, RenderTexture targetTexture,  CharacterPreviewInputEventBus inputEventBus);
+    }
+
+    public class CharacterPreviewFactory : ICharacterPreviewFactory
```

Comment:
You should create this class from the plugin and pass it as a dependency to any descendants such as `BackpackControler`. Otherwise, this factory is not reusable by other places

Reply:
Good point! fixed!